kivymd/vendor/circularTimePicker/__init__.py

```diff
@@ -415,8 +414,7 @@ def on_selected(self, *a):
             self._center_dot.size = csize
 
     def pos_for_number(self, n):
-        """Returns the center x, y coordinates for a given number.
-        """
+        """Returns the center x, y coordinates for a given number."""
 
         if self.items == 0:
             return 0, 0
```

Comment:
Updated by pre-commit of github actions. 8994a39cd2b1050aa6b8342b533d148cbe2e065a